packages/cli/src/utils/platform.ts

```diff
@@ -1,11 +1,15 @@
-import { Commands, unknownCommand } from '@prisma/internals'
+import Debug from '@prisma/debug'
+import { Commands, getCommandWithExecutor, unknownCommand } from '@prisma/internals'
 import fs from 'fs-extra'
+import { green } from 'kleur/colors'
 import fetch, { Headers } from 'node-fetch'
 import path from 'path'
 import XdgAppPaths from 'xdg-app-paths'
 
 import { getInstalledPrismaClientVersion } from './getClientVersion'
 
+const debug = Debug('prisma:platform')
```

Comment:
```suggestion
const debug = Debug('prisma:cli:platform')
```
Would be better I think, we already have one like this. (and feel free to fix it for Studio later if you want, currently it's `prisma:studio` and would be better as `prisma:cli:studio`)